src/helpers/helpers.sh

```diff
@@ -3,7 +3,7 @@
 
 current_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
-export PROGRAM="$(echo "$0" | grep -o "[^/]*$")"
+export PROGRAM="$(echo "$0" | sed 's/\\/\//g' | sed 's/://g'| grep -o "[^/]*$")"
```

Comment:
Can we extract the logic that is converting windows styles paths into unix styles paths into a function so the logic only lives in one place?